openhands/runtime/builder/docker.py

```diff
@@ -197,8 +195,13 @@ def image_exists(self, image_name: str, pull_from_repo: bool = True) -> bool:
 
                 layers: dict[str, dict[str, str]] = {}
                 previous_layer_count = 0
+                _splited_image_name = image_name.split(':')
+                image_repo = _splited_image_name[0]
+                image_tag = (
+                    _splited_image_name[1] if len(_splited_image_name) > 1 else None
```

Comment:
IF the image name doesn't have a ":" in it, this will run into an error (index).

Reply:
ooof good catch

Reply:
resolved in the latest commit!